src/frontend/screens/Settings/components/IgnoreGameUpdates.tsx

```diff
@@ -6,14 +6,14 @@ import SettingsContext from '../SettingsContext'
 
 const IgnoreGameUpdates = () => {
   const { t } = useTranslation()
-  const { isDefault } = useContext(SettingsContext)
+  const { isDefault, gameInfo } = useContext(SettingsContext)
 
   const [ignoreGameUpdates, setIgnoreGameUpdates] = useSetting(
     'ignoreGameUpdates',
     false
   )
 
-  if (isDefault) {
+  if (isDefault || gameInfo?.runner === 'hyperplay') {
```

Comment:
this prevents users from disabling auto updates on a per game basis for HyperPlay games? Is this desirable? Feels like we're taking away a feature some lower bandwidth users might desire

Reply:
That is what I understood from our meeting last week.
Because was said that the users should not be able to play an old version of a game and the update is mandatory.
This is the point of this PR in the end. 
If the users enable this option they will never receive any update notification for the game. If they forget to turn this off they will stuck in an old version.

@jacobc-eth @awantoch what do you guys think about it?

Reply:
just to clarify this feature makes it opt-out on a per-game basis right @flavioislima ?

Reply:
if it's disabling auto-updates per game, I think that's more desirable since other games would still be in sync more frequently and it would reduce the developer friction with keeping their players on the latest version

Reply:
I believe alec is saying ` just leave it but disabled by default (that is the current behavior)` but I will let him clarify

Reply:
Yeah I think so, basically just auto-update everything by default, but option to disable per game

Reply:
ok so I won't remove the setting for now, but we should get back to it if the users keep playing old versions of games.